### DISCUSSION TITLE
How to reuse opened file

### DISCUSSION THREAD

**Discussion by urayoru113** [nvim-tree/nvim-tree.lua#2644] [Q&A]
I opened a file named `foo.txt`.

When I opened the same file by pressing `t`.

It would open a new tab like:
`| foo.txt | (duplicated) foo.txt |`

How can I just jump to the opened file without opening a new tab?

**Comment by gegoune** [comment]
There is not enough information here to help you. What did you bind `t` to? Also what you are seeing, is it some plugin displaying your opened buffers?

**Reply by urayoru113** [reply]
Okey, I disabled all my plugins.

I opened a file named `foo.txt`
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/3b0b2976-a93d-4a6c-96b6-cbb3d9ae1634)

Here I press `<c-t>` which is binding to `Open: New tab` by default settings
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/988a96f9-eee9-4637-b56b-b1a777259d71)

Then there is an duplicted file here
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/5032215f-5ddc-46f5-8446-f267e1b9af5a)

My question is how can I just make the cursor jump to opened `foo.txt` rather than opening a new `foo.txt`

**Reply by urayoru113** [reply]
I mean if I open a lot of tabs, I want to make the cursor jump to opened file rather than opening a new one

**Reply by gegoune** [reply]
Read up what tabs in vim/neovim are exactly, they are not the same as, let's say, browser tabs. You are not opening the same file again, but creating new tab with the same buffer displayed in single window in there.
There is no issue here, if you don't want to create new tab then don't use `<C-t>` but `<CR>` for example.

**Reply by urayoru113** [reply]
Sorry, I should explain my problem more clear.

If I open two files `foo.txt` and `bar.txt`, and my cursor is on `foo.txt` of the nvimtree

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/89f85949-4b94-433d-9b5a-c4f6a1be4d3c)

If I press `<c-t>` nvimtree will open new `foo.txt`,
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/710dc07e-6628-4176-b2d8-787fd22f81fe)


 If I press `enter`, nvim tree will replace `bar.txt` to `foo.txt`

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/69f82297-dbee-4c20-80c4-ba53867c2347)

How to jump to opened `foo.txt` with the cursor on it.
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/841cce98-4eba-4d49-b16e-0547e5c5341f)

Did nvimtree provide any solution or I have to write a custom autocmd?

It seems to relative to this issue #1687

**Reply by gegoune** [reply]
It's more vim usage question, really, you can start by reading `:h buffers`. (You probably don't want to open buffers in new tab, but in existing windows instead, which is set by default to `<CR>`.)
Have a look at `:h :ls` and `:h :b`.

**Reply by alex-courtis** [reply]
> (You probably don't want to open buffers in new tab, but in existing windows instead, which is set by default to <CR>.)

Indeed. Tabs in vim are for groupings of related windows, not for individual files like most IDEs.

See #1598 for related discussion.
